Makefile.dep

```diff
@@ -629,6 +629,20 @@ ifneq (,$(filter spiffs,$(USEMODULE)))
     USEMODULE += mtd
 endif
 
+ifneq (,$(filter fatfs_vfs,$(USEMODULE)))
+    USEPKG += fatfs
+    USEMODULE += vfs
+
+    ifeq ($(BOARD),native)
+      USEMODULE += fatfs_diskio_native
+      FATFS_DISKIO_NATIVE_DEFAULT_FILE ?= \"riot_fatfs_disk.img\"
+    else
+      USEMODULE += fatfs_diskio_sdcard_spi
+      USEMODULE += auto_init_storage
+    endif
```

Comment:
> Yes but no... I think the decision which device should actually be used by the FS should be made "as near as possible to the application". So for now (with the current shortcomings of the diskio layer) I'd like to keep this workaround there and have it removed from this single place once the diskio becomes almighty with the help of mtd ;)

I might have been unclear on this point. What I meant was: the actual *drive* (or storage or whatever) information should reside with the board. An application user (read also developer) typically doesn't want to and doesn't need to know if a drive is available via SPI, I2C, IDE, or SATA, or as a file that just emulates a storage. they just need to know: there is a drive. Compare Linux where you just get a file `/dev/sdX` as soon as you plug-in a drive regardless of the interface (technically there are differences between the naming of non-SCSI HDDs, SD cards, etc., but those are only names and as different distributions show: highly software dependent and in theory unifiable to `/dev/storageX`).